Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -255,6 +255,12 @@ public void Set(MixedRealityRaycastHit hit, RayStep ray, int rayStepIndex, float
             public void Set(RaycastResult result, Vector3 hitPointOnObject, Vector4 hitNormalOnObject, RayStep ray, int rayStepIndex, float rayDistance)
             {
                 raycastHit = default(MixedRealityRaycastHit);
+                raycastHit.point = hitPointOnObject;
+                raycastHit.normal = hitNormalOnObject;
+                raycastHit.distance = rayDistance;
+                raycastHit.transform = result.gameObject.transform;
+                raycastHit.raycastValid = false;
+
```

Comment:
Should these variables also be set for a physics raycast above? Also, out of interest, why `raycastHit.raycastValid = false;`?

Reply:
I set this to false for now because I can't populate all the fields in case of the graphics raycast - I wanted to communicate in some way that the data is not complete. (todo: add comment)

in the case of the physics raycast the fields are already all populated by the scene query - we pass the mixedrealityraycasthit we've received from the scene query into the set method above. 

To give more context on this:
there's two ways I can fix this bug - one of them is I change the interface of the gaze provider to take a PointerHitResult instead of a MixedRealityRaycastHit to be able to pass in all the info I need to handle both the graphics and physics hit.
Or the second solution is to fill out the MixedRealityRaycastHit entry for a graphicsraycast manually and that way pass the info along.
The first one implies a breaking change the second one doesn't require any release notes / migration instructions but is not as clean as the interface change.
I discussed this yesterday with @keveleigh and wanted to give the solution without breaking change a go - if people thing we should rather go with the interface change that's fine as well :)